client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -94,9 +94,6 @@ public ClientConfig build() {
                 controllerURI = URI.create("tcp://localhost:9090");
             }
             extractCredentials();
-            if (credentials == null) {
```

Comment:
It seems fine and even desirable to remove this message, but one question I have is whether we are warning the user when security is disabled. We should have a WARN message to make sure a user is not running with security disabled by accident.

Reply:
If the server is running with `auth` enabled, the client application sees an exception indicating authentication failure in both these cases: 
* Credentials are invalid 
* Credentials are missing

So, the client will get an exception if the credentials are missed by accident. (On a different note, the indication - i.e., the exception name and the message - has significant room for improvement, but I'll leave that for another PR.)

The problem that we are addressing here is that the logs on the client side indicate that credentials are missing even in these cases: 
* The client application has actually supplied them (due to some internal calls that happen under the hoods in the client library). 
* The client application is talking to a cluster where `auth` is disabled. In real applications, `auth` should be enabled, and therefore it shouldn't be a real issue in those cases. But, when using a cluster for development/testing or a standalone mode cluster, application developers can get confused by this message. 

I've faced these issues and have got confused on several occasions. I've also seen people pointing to this message when something else has gone wrong elsewhere. :)